docs/learn/intro/01-why-app-specific.md

```diff
@@ -12,20 +12,28 @@ This document explains what application-specific blockchains are, and why develo
 
 Application-specific blockchains are blockchains customized to operate a single application. Instead of building a decentralized application on top of an underlying blockchain like Ethereum, developers build their own blockchain from the ground up. This means building a full-node client, a light-client, and all the necessary interfaces (CLI, REST, ...) to interact with the nodes.
 
-```text
-                ^  +-------------------------------+  ^
-                |  |                               |  |   Built with Cosmos SDK
-                |  |  State-machine = Application  |  |
-                |  |                               |  v
-                |  +-------------------------------+
-                |  |                               |  ^
-Blockchain node |  |           Consensus           |  |
-                |  |                               |  |
-                |  +-------------------------------+  |   CometBFT
-                |  |                               |  |
-                |  |           Networking          |  |
-                |  |                               |  |
-                v  +-------------------------------+  v
+```mermaid
+flowchart TD
+    subgraph Blockchain_Node[Blockchain Node]
+        subgraph SM[State-machine]
+            direction TB
+            SM1[Cosmos SDK]
+        end
+        subgraph Consensus[Consensus]
+            direction TB
+        end
+        subgraph Networking[Networking]
+            direction TB
+        end
+    end
+
+    SM <-->|Interaction| Consensus
```

Comment:
"Interaction" shows up in the rendered diagram. Do we want that?

<img width="462" alt="Screenshot 2024-05-31 at 12 11 38 PM" src="https://github.com/cosmos/cosmos-sdk/assets/14063057/7ccc6a25-6edf-4eea-b15e-c186864da60e">

Reply:
can remove, it was generated.